File: java-sdk/lib/src/main/java/models/shared/MetadataQueryResults.java
package .models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * MetadataQueryResults
 * A page of files and folders that matched the metadata query.
**/
public class MetadataQueryResults {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("entries")
    public Object[] entries;
    public MetadataQueryResults withEntries(Object[] entries) {
        this.entries = entries;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("limit")
    public Long limit;
    public MetadataQueryResults withLimit(Long limit) {
        this.limit = limit;
        return this;
    }
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("next_marker")
    public String nextMarker;
    public MetadataQueryResults withNextMarker(String nextMarker) {
        this.nextMarker = nextMarker;
        return this;
    }
}

File: sdks/python/src/boxsdk/models/shared/metadataqueryresults.py
import dataclasses
from typing import Any,Optional
from dataclasses_json import dataclass_json
from boxsdk import utils


@dataclass_json
@dataclasses.dataclass
class MetadataQueryResults:
    r"""MetadataQueryResults
    A page of files and folders that matched the metadata query.
    """
    
    entries: Optional[list[Any]] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('entries') }})
    limit: Optional[int] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('limit') }})
    next_marker: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('next_marker') }})
    
